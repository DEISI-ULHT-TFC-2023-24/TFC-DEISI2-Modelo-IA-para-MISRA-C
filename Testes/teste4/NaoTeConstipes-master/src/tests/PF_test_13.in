confstart
a
x
b
x
b
x
s
