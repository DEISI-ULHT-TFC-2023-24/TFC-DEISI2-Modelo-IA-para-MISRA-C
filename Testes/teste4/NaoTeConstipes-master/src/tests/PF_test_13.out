+--------------------------------------------------------+
|         Nao Te Constipes - configs                     |
+--------------------------------------------------------+
| confadd <numero>    - adicionar casa segura            |
| confsave <nome>     - gravar ficheiro de casas seguras |
| confboard <numero>  - mostrar o tabuleiro              |
| confstart <numero>  - iniciar o jogo                   |
| confmenu            - imprimir menu                    |
+--------------------------------------------------------+
>+-- 1--++-- 2--++-- 3--++-- 4--++-- 5--++-- 6--++-- 7--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
+-- 0--+                                        +-- 8--+
| abcd |                                        |      |
| **** |                                        | **** |
|      |                                        | wxyz |
+------+                                        +------+
+--15--++--14--++--13--++--12--++--11--++--10--++-- 9--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
+------------------------------------+
|         Nao Te Constipes           |
+------------------------------------+
| <id do peao> (abcd, xyzw)          |
| s - sair                           |
| h - imprimir menu                  |
+------------------------------------+
------ Jogador 1 ------
Jogador lancou dados com valor 7 
>+-- 1--++-- 2--++-- 3--++-- 4--++-- 5--++-- 6--++-- 7--+
|      ||      ||      ||      ||      ||      || a    |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
+-- 0--+                                        +-- 8--+
|  bcd |                                        |      |
| **** |                                        | **** |
|      |                                        | wxyz |
+------+                                        +------+
+--15--++--14--++--13--++--12--++--11--++--10--++-- 9--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
------ Jogador 2 ------
Jogador lancou dados com valor 6 
>+-- 1--++-- 2--++-- 3--++-- 4--++-- 5--++-- 6--++-- 7--+
|      ||      ||      ||      ||      ||      || a    |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
+-- 0--+                                        +-- 8--+
|  bcd |                                        |      |
| **** |                                        | **** |
|      |                                        | w yz |
+------+                                        +------+
+--15--++--14--++--13--++--12--++--11--++--10--++-- 9--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||  x   ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
------ Jogador 1 ------
Jogador lancou dados com valor 8 
>+-- 1--++-- 2--++-- 3--++-- 4--++-- 5--++-- 6--++-- 7--+
|      ||      ||      ||      ||      ||      || a    |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
+-- 0--+                                        +-- 8--+
|   cd |                                        |  b   |
| **** |                                        | **** |
|      |                                        | w yz |
+------+                                        +------+
+--15--++--14--++--13--++--12--++--11--++--10--++-- 9--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||  x   ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
------ Jogador 2 ------
Jogador lancou dados com valor 6 
>+-- 1--++-- 2--++-- 3--++-- 4--++-- 5--++-- 6--++-- 7--+
|      ||      ||      ||      ||      ||      || a    |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||  x   ||      ||      ||      |
+------++------++------++------++------++------++------+
+-- 0--+                                        +-- 8--+
|   cd |                                        |  b   |
| **** |                                        | **** |
|      |                                        | w yz |
+------+                                        +------+
+--15--++--14--++--13--++--12--++--11--++--10--++-- 9--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
------ Jogador 1 ------
Jogador lancou dados com valor 6 
>+-- 1--++-- 2--++-- 3--++-- 4--++-- 5--++-- 6--++-- 7--+
|      ||      ||      ||      ||      ||      || a    |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||  x   ||      ||      ||      |
+------++------++------++------++------++------++------+
+-- 0--+                                        +-- 8--+
|   cd |                                        |      |
| **** |                                        | **** |
|      |                                        | w yz |
+------+                                        +------+
+--15--++--14--++--13--++--12--++--11--++--10--++-- 9--+
|      ||  b   ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
------ Jogador 2 ------
Jogador lancou dados com valor 5 
>+-- 1--++-- 2--++-- 3--++-- 4--++-- 5--++-- 6--++-- 7--+
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
+-- 0--+                                        +-- 8--+
| a cd |                                        |      |
| **** |                                        | **** |
|      |                                        | wXyz |
+------+                                        +------+
+--15--++--14--++--13--++--12--++--11--++--10--++-- 9--+
|      ||  b   ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
|      ||      ||      ||      ||      ||      ||      |
+------++------++------++------++------++------++------+
------ Jogador 1 ------
Jogador lancou dados com valor 5 
>Fim do jogo
